src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Ball ball = new Ball();
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
Think about which variables should be local in the method and which should be class-level

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery

Reply:
Not fixed